mklove/modules/configure.base

```diff
@@ -521,10 +521,12 @@ function mkl_generate {
 
     # Generate config.h
     mkl_write_h "// Automatically generated by $0 $*"
-    mkl_write_h "#pragma once"
+    mkl_write_h "#ifndef _CONFIG_H_"
+    mkl_write_h "#define _CONFIG_H_ 1"
```

Comment:
No need for defining it to 1, just an empty define is more standard.
This goes for all of these

Reply:
I feel like I commonly see both, but it doesn't matter to me.